docs/kb/api-version-3/querying-visits.md

```diff
@@ -1,53 +0,0 @@
-
```

Comment:
Is this file not used?

Reply:
They are not linked from anywhere. The querying-visits.md files appear to be old files and the content is duplicated in visits.md.